README.md

```diff
@@ -204,6 +206,30 @@ zu.SPR( numberArraySchema.safeParse( 'foo' ) ).error?.issues[ 0 ].message
 // 'Expected number, received nan'
 ```
 
+### JSON
+
+Parse a JSON object literal:
+```ts
+import { zu } from 'zod_utilz'
+const schema = zu.json()
+schema.parse(false) // false
+schema.parse(8675309) // 8675309
+schema.parse({ a: 'deeply', nested: [ 'JSON', 'object' ] })
+// { a: 'deeply', nested: [ 'JSON', 'object' ] }
+```
+
+### jsonString
```

Comment:
What happens if we want to add a `jsonToString` type in the future that's just `zu.json().transform(o => JSON.stringify(o))`? Would that be called `jsonToString`?

Should this be called `stringToJSON`?

Reply:
> What happens if we want to add a `jsonToString` type in the future that's just `zu.json().transform(o => JSON.stringify(o))`? Would that be called `jsonToString`?

I'm all for this. jsonToString sounds good to me.

---

> Should this be called `stringToJSON`?
I think that `jsonString` should be changed to `stringToJSON`, because it is doing a transform.